openstf/model/confidence_interval_applicator.py

```diff
@@ -7,19 +7,29 @@
 import numpy as np
 import pandas as pd
 from scipy import stats
+from sklearn.base import RegressorMixin
+
+from openstf.enums import MLModelType
 
 MINIMAL_RESOLUTION: int = 15  # Minimal time resolution in minutes
 
 
 class ConfidenceIntervalApplicator:
-    def __init__(self, model):
-        self.confidence_interval = model.confidence_interval
+    def __init__(self, model: RegressorMixin, forecast_input_data: pd.DataFrame):
```

Comment:
Is er eigenlijk niet een nog hoge base class van sklearn? Iets van een `Estimator` oid. Voor nu denk wel prima verder.

Reply:
The Base estimator also exists: https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html#sklearn.base.BaseEstimator. The type of the regressor is a combination of the regressor mixin and the base eatimator